module/data/shared/roll-config-field.mjs

```diff
@@ -18,6 +18,11 @@ export default class RollConfigField extends foundry.data.fields.SchemaField {
   constructor({roll={}, ability="", ...fields}={}, options={}) {
     const opts = { initial: null, nullable: true, min: 1, max: 20, integer: true };
     fields = {
+      ability: ( ability === false ) ? null : new StringField({
```

Comment:
You won't need the check here, because the `...fields` at the end will replace this with `false` for you.

Reply:
How so? We destructure the property because it should be `""` if omitted.

Reply:
Aw, yeah, I forgot you are also using `ability` to set the default value as well as to remove the field. Ignore me.